dist/package.sh

```diff
@@ -78,12 +78,23 @@ targets=(
     sdp/sdp.js
 )
 
+
 for target in "${targets[@]}"; do
-    install -D "node_modules/$target" "$DIR/node_modules/$target"
+
+	if [[ "$OSTYPE" == "darwin"* ]]; then
+		ditto "node_modules/$target" "$DIR/node_modules/$target"
+    else
+		install -D "node_modules/$target" "$DIR/node_modules/$target"
+    fi
 done
 
+
 echo "+ Update version number..."
-sed -i "s/\[\[VERSION\]\]/${VERSION}/g" $DIR/index.html $DIR/troubleshoot/index.html $DIR/dist/app.js $DIR/version.txt
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	    gsed -i "s/\[\[VERSION\]\]/${VERSION}/g" $DIR/index.html $DIR/troubleshoot/index.html $DIR/dist/app.js $DIR/version.txt
```

Comment:
What's the problem with regular `sed`? I don't think the user should have to install any additional build dependencies to run the build script.

Reply:
There are some differences between sed on versions of Unix using the GNU tools and sed on versions based on freeBSD like macOS (https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed).
I'm not sure what needs to be changed to get it running on both platforms and thought that installing gsed is only a minor inconvenience.

Reply:
This should work with regular `sed` on macOS if you use `sed -i ''` instead of `sed -i`.